src/rules/seczetta-grab-risk-score.js

```diff
@@ -0,0 +1,223 @@
+/* global configuration, auth0 */
+
+/**
+ * @title seczettaGrabRiskScore
+ * @overview Grab the risk score from SecZetta to use in the authentication flow
+ * @gallery true
+ * @category multifactor"
+ * 
+ * 
+ * **Required configuration** (this Rule will be skipped if any of the below are not defined):
+ *
+ *    - `SECZETTA_API_KEY` API Token from your SecZetta tennant
+ *    - `SECZETTA_BASE_URL` URL for your SecZetta tennant
+ *    - `SECZETTA_ATTRIBUTE_ID` the id of the SecZetta attribute you are searching on (i.e personal_email, user_name, etc.)
+ *    - `SECZETTA_PROFILE_TYPE_ID' the id of the profile type this user's profile
+ *
+ * **Optional configuration:**
+ *
+ *    - `SECZETTA_AUTHENTICATE_ON_ERROR` Choose whether or not the rule continues to authenticate on error
+ *    - `SECZETTA_RISK_KEY` The attribute name on the account where the users risk score is stored
+ *    - `SECZETTA_ALLOWABLE_RISK` Set to a risk score integer value above which MFA is required
+ *    - `SECZETTA_MAXIMUM_ALLOWED_RISK` Set to a maximum risk score integer value above which login fails.
+ *    
+ * **Helpful Hints**
+ *    - The SecZetta API documentation is located here: https://{{SECZETTA_BASE_URL}}/api/v1/
+ *
+ * @param {object} user
+ * @param {object} context
+ * @param {function} callback
+ */
+async function seczettaGrabRiskScore(user, context, callback) {
+  
+  if (!configuration.SECZETTA_API_KEY || !configuration.SECZETTA_BASE_URL) {
+    console.log("Missing required configuration. Skipping.");
+    return callback(null, user, context);
+  }
+
+  const userUtils = new require(
+    "@auth0/rule-utilities@0.2.0"
+  );
+
+  const axios = require("axios@0.19.2");
+      
+  //As of right now (12/15/2020), SecZetta has to call 2 apis to grab the risk score. One to grab the profile
```

Comment:
It might be better to document this in the installation guide instead of here. It's more likely to be seen there.